subprojects/file-watching/src/main/java/org/gradle/internal/watch/vfs/BuildLifecycleAwareVirtualFileSystem.java

```diff
@@ -32,7 +32,12 @@
     /**
      * Called when the build is started.
      */
-    void afterBuildStarted(boolean watchingEnabled, VfsLogging vfsLogging, WatchLogging watchLogging, BuildOperationRunner buildOperationRunner);
+    void afterBuildStarted(WatchMode watchingEnabled, VfsLogging vfsLogging, WatchLogging watchLogging, BuildOperationRunner buildOperationRunner);
```

Comment:
:o: Should probably rename the parameter to `watchMode`

Reply:
Good point.